apps/store/themes/store/js/list_assets.js

```diff
@@ -377,3 +590,12 @@ $(window).load(function () {
         document.getElementById("categoryDropDown").title = document.getElementById("categoryDropDown").text.trim()
     }
 });
+
+$( document ).ready(function() {
```

Comment:
We do not need to assign it to another variable as store.assetAvailability is already a global variable